File: tests/tools/read_many_files/test_data/sample.py
"""
Sample Python file for testing ReadManyFiles tool.
"""

def hello_world():
    """Print hello world message"""
    print("Hello, World!")

class Calculator:
    """Simple calculator class"""
    
    def __init__(self):
        self.result = 0
    
    def add(self, x, y):
        """Add two numbers"""
        return x + y
    
    def subtract(self, x, y):
        """Subtract two numbers"""
        return x - y

if __name__ == "__main__":
    hello_world()
    calc = Calculator()
    print(f"2 + 3 = {calc.add(2, 3)}")

File: tests/tools/read_many_files/test_data/sample.js
/**
 * Sample JavaScript file for testing ReadManyFiles tool.
 */

function greetUser(name) {
    return `Hello, ${name}!`;
}

class MathUtils {
    static add(a, b) {
        return a + b;
    }
    
    static multiply(a, b) {
        return a * b;
    }
}

// Export for Node.js
if (typeof module !== 'undefined' && module.exports) {
    module.exports = { greetUser, MathUtils };
}

console.log(greetUser("World"));
console.log("5 + 3 =", MathUtils.add(5, 3));
